packages/core/src/components/Table/TableHeader/TableHeader.module.scss

```diff
@@ -3,10 +3,22 @@
   grid-template-columns: var(--table-grid-template-columns);
   position: sticky;
   top: 0;
-  z-index: 1;
+  z-index: 2;
   background-color: inherit;
+  min-width: 100%;
```

Comment:
What if the table is smaller than full width? Would it stretch?

Reply:
it would stretch to fit-content, but at minimum would fit to 100%

Reply:
if the table is in a smaller container it would behave the same, just relative to the smaller container

Reply:
Oh it's not the header? the table will fit content?

Reply:
both the header row and body row